tests/run.sh

```diff
@@ -41,64 +41,68 @@ popd >/dev/null
 
 ############################################################
 
-echo "Testing import-output-file"
-pushd "$base_dir"/import_only >/dev/null
+# TODO: enable this test again when using CI with Xcode 16.3+
```

Comment:
can you fix the tests now to verify they work locally as we'd expect

Reply:
hm from what i understood these tests are failing _in_ index-import with:

```
Testing import-output-file
error: failed to read unit file input/v5/units/input.c.o-3TYUTGSZ8FF3H -- Failed opening 'input/v5/units/input.c.o-3TYUTGSZ8FF3H': No such file or directory
```

I think this is because the has expects the Swift 6.1 one but CI is using Xcode 15.x. Is it possible to fix these, GitHub doesnt have Xcode 16.3 available.

See: https://github.com/MobileNativeFoundation/index-import/actions/runs/14232916900/job/39886942224

Reply:
yea i mean fix the script and disable the github action, so we can at least test locally

Reply:
ahhh yeah good idea, let me do that

Reply:
Tests pass locally now when using Xcode 16.3, disabled it in CI until github supports it